cron/src/tests/test_message_service.py

```diff
@@ -55,12 +49,20 @@ def test_compose_messages_to(self, test_data):
             m['To'] == 'borrower_surname_2 ' +
                        'borrower_name_2 <borrower_email_2@example.com>']) == 1
 
+    def test_compose_messages_to_admin(self, test_data):
+        sender = 'foo <foo@example.com>'
+        service = MessageService(sender)
+        messages = service.compose_admin_messages('', test_data)
+
+        for m in messages:
+            assert m['To'] == 'test'
```

Comment:
Where do make an arrangement for 'To' to be 'test'? It it comes from .env file, then it isn't unit test anymore.